### DISCUSSION TITLE
root@minios:~#  on Minios bookworm-flux minimum istallation

### DISCUSSION THREAD

**Discussion by mirubas1** [minios-linux/minios-live#31] [Q&A] [answered]
Hi friends. I am trying to install Minios bookworm-flux minimum on a PI4 PC and at a certain point of he process appears root@minios:~#. I need to know what to do to get a successful Minios installation and use this OS. Thanks in advance

**Comment by FershoUno** [comment, marked as answer]
PI4 = Raspberry PI 4?

If this is correct, then it is not possible to install MiniOS, the only architectures it supports are i386 and x86_64
